### ISSUE TITLE
installation support

### ISSUE THREAD

**Issue by ZLangJIT** [LekKit/RVVM#156]
does rvvm currently support `make install` or `cmake --install` (eg, for installation into rootfs's or system wide installs) ?

**Comment by LekKit** [comment]
Yes it is possible to install using `make install`. It is usually used for packaging system purposes (I.e. generate a distro package using `make install DESTDIR=...`), but it is possible to just copy into the rootfs directly.

**Comment by ZLangJIT** [comment]
i assume it installs headers to include/rvvm ?

**Comment by LekKit** [comment]
Yes.
